lib/catalog/data_driven_form_validator.rb

```diff
@@ -75,7 +84,7 @@ def check_validators(schema_validators)
 
       def check_options(schema_fields)
         schema_fields.map do |field|
-          next unless field[:component] == "select-field"
+          next unless field[:component] == "select-field" || field[:component] == "select" 
```

Comment:
@Hyperkid123 Will this eventually change back to a single `unless` condition after you complete the move to DDFv2?  As a general rule (I think rubocop complains about this) we try to avoid `unless` statements with more than one condition.  I would not mind this as much if it is temporary, as I suspect that it is.

Reply:
Yes eventually there will be just select. Actualy ddfv1 has multiple different strings for a number of components. One of the silly mistakes we have fixed.